StudCity.Frontend/src/services/userService.js

```diff
@@ -25,25 +24,25 @@ export class UserService {
 
 	async getUserById(id) {
 		try {
-			const { data } = await this.service.get(`${this.apiUrl}/${id}`)
+			const { data } = await this.service.get(`${this.apiUrl}/${id}`);
 			return data;
-		}
-		catch (err) {
-			showAlert("User not found", "error");
+		} catch (err) {
+			showAlert('User not found', 'error');
 		}
 	}
 
-	async getUsers({ page = 1, pageSize = 10, searchWord = ''  }) {
+	async getUsers({ page = 1, pageSize = 10, searchWord = '' }) {
 		try {
-			const { data } = await this.service.get(`${this.apiUrl}/search/?page=${page}&pageSize=${pageSize}&searchWord=${searchWord}`);
+			const { data } = await this.service.get(
+				`${this.apiUrl}/search/?page=${page}&pageSize=${pageSize}&searchWord=${searchWord}`
```

Comment:
you can also wright it like this:
this.service.get(`${this.apiUrl}/search`, {
   params: { page, pageSize, searchWord }
});
this syntax is usefull cause it does additional parsing, for example if searchWord consists of a few words devided by spaces it will automatically replace a space with '%20' which is used for encoding when sending filter request from client to the server